tests/profile.spec.js

```diff
@@ -60,25 +60,36 @@ test("Link navigates", async ({ page }) => {
 
   //   // 2. get a link and href
   const anchorTags = await page.$$("a");
```

Comment:
I wonder if this is too broad as there are other links on the page 🤔

Reply:
I think this use has no links and an app link is being visited

<img width="1667" alt="Screenshot 2023-08-25 at 08 24 25" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/cdf17f1f-61e8-42c3-be5e-69c313df7ad1">